iOS/Source/MainController.swift

```diff
@@ -42,7 +42,7 @@ class MainController: UIViewController {
         self.locationTracker.locateIfPossible()
         self.updateInterface()
 
-        Timer.scheduledTimer(timeInterval: 1.0, target: self, selector: #selector(self.updateInterface), userInfo: nil, repeats: true)
+        Timer.scheduledTimer(timeInterval: 30, target: self, selector: #selector(self.updateInterface), userInfo: nil, repeats: true)
```

Comment:
Any particular reason to bump this up as high as 30 instead of just making it less than once a second?

Reply:
Wouldn't that be the opposite of what we need? Instead of refreshing the screen every second, every 30s is good enough without just depleting battery.

Reply:
Another option is to keep it at 1s intervals, but only update the screen if the minute changed. 🤔

Reply:
I meant more like bumping it to 5 seconds instead of 30, but that makes sense. I suppose exact minute accuracy is not what we're going for here.